client/src/Layout.js

```diff
@@ -1,20 +1,56 @@
-import { Outlet, Link } from "react-router-dom";
+import "./App.css";
```

Comment:
All css should be imported in one component only. Please move all css imports to App.js.